tests/update_testapps_dependencies.go

```diff
@@ -0,0 +1,80 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+)
+
+func main() {
+	var files []string
+
+	root := "./apps"
+	_ = filepath.Walk(root, func(path string, info os.FileInfo, err error) error {
+		if err != nil {
+			panic(err)
+		}
+		if info.IsDir() {
+			_, err := os.Stat(path + "/go.mod")
+			if !os.IsNotExist(err) {
+				files = append(files, path)
+			}
+		}
+		return nil
+	})
+
+	testsPath, _ := os.Getwd()
+	goExecutable, _ := exec.LookPath("go")
+	for _, file := range files {
+		os.Chdir(file)
+		_, err := RunCmdAndWait(goExecutable, "get", "-u", "github.com/dapr/dapr@master")
+		if err != nil {
+			panic(err)
+		}
+		fmt.Println("Successfully modified dapr dependencies for: ", file)
```

Comment:
nit : start with lowercase only..we are using all traces that way..

Reply:
fixed